### ISSUE TITLE
No info about new version available since PS 8.0.0

### ISSUE THREAD

**Issue by SharakPL** [PrestaShop/PrestaShop#35672]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

Since https://github.com/PrestaShop/PrestaShop/pull/26239 there's no information about new version available on admin dashboard. Removed section was hijacked by https://github.com/PrestaShopCorp/ps_mbo which now only serves blog news and addons connect feature. No info about new version ☹️

### Expected behavior

PS 1.7:

![obraz](https://github.com/PrestaShop/PrestaShop/assets/5007456/e94a8358-642f-4421-8e36-cb430c0d27db)

### Steps to reproduce

Install the shop (any version from range 8.0.0 - 8.1.3) and check if you have info about version 8.1.4 available anywhere.

### PrestaShop version(s) where the bug happened

8.1.x

### PHP version(s) where the bug happened

8

### If your bug is related to a module, specify its name and its version

https://github.com/PrestaShopCorp/ps_mbo

**Comment by florine2623** [comment]
Hello @SharakPL ,

If I'm not mistaken, this was actually a feature for PS 8.

@MatShir could you confirm ?

Thanks!

**Comment by SharakPL** [comment]
Yes, to move some parts into ps_mbo. But why remove notification about new version available completely? Even with ps_mbo installed there's no info about it. The only way to know there's new version is to check configuration of autoupgrade module. What was so wrong about showing it directly on the dashboard?

**Comment by MatShir** [comment]
The feature needs to be backported to autoupgrade module or somewhere else

**Comment by Quetzacoalt91** [comment]
Update Assistant provides another way to display new versions available for update.

![Image](https://github.com/user-attachments/assets/626a0815-29bc-4e82-a1a7-81bf32616932)

You can find it after installing the version 7.1.

**Comment by SharakPL** [comment]
@Quetzacoalt91 I've never heard about an update assistant. What module is this?

**Comment by Quetzacoalt91** [comment]
This is the new name of [`autoupgrade`](https://github.com/PrestaShop/autoupgrade) responsible of updating PrestaShop stores.
